tests/lua-tests/src/SpritePolygonTest/SpritePolygonTest.lua

```diff
@@ -690,7 +690,7 @@ end
 
 function SpritePolygonPerformanceTestDynamic:initIncrementStats()
     self._incVert = self._info:getVertCount()
-    self._incTri  = self._info:getTriaglesCount()
```

Comment:
@mogemimi  I see, lua use auto bindings :smile:  Thanks for the link. I'm fine with this PR now.